verifier.sh

```diff
@@ -472,12 +479,12 @@ while [ $# -gt 0 ]; do
             #    usage 1
             #fi
             ACTION="$1"
-            devtest_run $(echo "$2" | sed 's@.*/@@g') "$3" "$4"
+            devtest_run $(echo "$2" | sed 's@.*/@@g') "$3" "$4" "$5"
             break
             ;;
          prodtest)
             ACTION="$1"
-            prodtest_run $(echo "$2" | sed 's@.*/@@g')
+            prodtest_run $(echo "$2" | sed 's@.*/@@g') "$3" "$4"
```

Comment:
at 391 you can see that `prodtest_run` function has 4 parameters but at line 487 you pass just three, please fix it and understand why tests are running correctly.

Reply:
Fixed with this https://github.com/gem/oq-platform2/pull/104/commits/47a04232fd81e64c668e33038c1b3b51c7449d54